extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1547,6 +1559,26 @@ export class AmpA4A extends AMP.BaseElement {
    */
   emitLifecycleEvent(unusedEventName, opt_extraVariables) {}
 
+  tryExecuteRealTimeConfig_() {
+    if (!!AMP.realTimeConfigManager) {
+      try {
+        return AMP.realTimeConfigManager(
+            this, this.getCustomRealTimeConfigMacros_());
+      } catch (err) {
+        dev().error(TAG, err);
```

Comment:
Combine into single dev error.  Could this actually throw?  Can we make it so it doesn't and in the event of an error just returns undefined?  I would want to know how it fails so that we can make it clear the impls should they decide to handle (e.g. not send the ad request)